### ISSUE TITLE
options=count

### ISSUE THREAD

**Issue by kzangeli** [Fiware/context.Orion-LD#430]
ETSI CIM has decided to change the 'count' URI parameter and modify it from `options` to its own URL parameter.

The new URI parameter will be called `count` and it needs a value of 'true' to be turned on.
`false` is default.

The response to `count`  is delivered in an HTTP header called `NGSILD-Results-Count`

**Comment by jason-fox** [comment]
There is now v2/LD incompatibility:

```console
curl -L -X GET 'http://localhost:1026/ngsi-ld/v1/entities/?type=TemperatureSensor&limit=0&count=true&q=controlledAsset==%22urn:ngsi-ld:Building:barn002%22' \
-H 'Link: <http://context-provider:3000/data-models/json-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
-H 'Accept: application/json'
```

Returns a *header* `NGSILD-Results-Count: 1`  ✅ 


```console
curl -L -X GET 'http://localhost:1026/v2/entities/?q=refProduct==urn:ngsi-ld:Product:001&options=count&attrs=type' 
```

Kills the context broker :x: &nbsp; - maybe related to #508

**Comment by kzangeli** [comment]
So, something seems to have broken.
The idea is for both of them to work, until we decide otherwise.
Taking a look right now. Should be an easy fix.

**Comment by kzangeli** [comment]
So, I tested this and it works just fine.
However, I don't use your exact data, so the test is not conclusive.
Please post the curl commands to create the entities before doing these queries, see if I can reproduce the error.

**Comment by kzangeli** [comment]
Fixed by PR #510
